### PR TITLE
fix: use config value for mobile column

### PR DESCRIPTION
**Pull request by MajdTaweel**

### CODE REVIEW

File: src/OTPBroker.php [hunk 1]
```diff
@@ -132,6 +132,7 @@ private function tokenExists(OTPNotifiable $user, string $token): bool
 
     private function makeNotifiable(string $mobile): OTPNotifiable
     {
-        return $this->userRepository->getModel()->make(['mobile' => $mobile]);
+        $mobileColumn = config('otp.mobile_column');
+        return $this->userRepository->getModel()->make([$mobileColumn => $mobile]);
```
Comment (mohammad-fouladgar) [inline src/OTPBroker.php:136]: Please add a new line above this line.

**Review by mohammad-fouladgar — COMMENTED** [review]
Thanks.

File: src/OTPBroker.php (hunk 1 above)
Comment (mohammad-fouladgar) [inline src/OTPBroker.php:137]: Please add a new line above this line.

Comment (MajdTaweel) [reply src/OTPBroker.php:137]: All done!

**Comment by MajdTaweel** [comment]
> Thanks.

My pleasure!

**Comment by mohammad-fouladgar** [comment]
> > Thanks.
> 
> My pleasure!

Welcome!

**Pull request merged**